src/HonzaBotner.Services/CvutAuthorizationService.cs

```diff
@@ -99,7 +99,7 @@ RolesPool rolesPool
 
             if (rolesGranted)
             {
-                Verification verification = new() { AuthId = authId, UserId = userId };
+                Verification verification = new() { AuthId = authId, UserId = userId, Username = person.Username };
```

Comment:
This should also be updated above for `// discord and auth -> update roles` section.